Small tweak to language and un-stacking the callouts (#139)
hashicorp/terraform-docs-agents: website/docs/partials/agents.mdx

> -> **Note:** If you are using HCP Terraform, your plan affects the number of agents you can use. Refer to [HCP Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.

Reviewer: ```suggestion
-> **Note:** Your HCP Terraform plan determines the number of agents you can use. Refer to [HCP Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
```
